Modules/Filtering/DisplacementField/include/itkDisplacementFieldTransform.hxx

```diff
@@ -383,10 +383,10 @@ DisplacementFieldTransform<TParametersValueType, VDimension>::VerifyFixedParamet
     const double coordinateTolerance = m_CoordinateTolerance * fieldSpacing[0];
     const double directionTolerance = m_DirectionTolerance;
 
-    std::ostringstream       sizeString;
-    std::ostringstream       originString;
-    const std::ostringstream spacingString;
-    const std::ostringstream directionString;
```

Comment:
@hjmjohnson FYI I spotted this bug because of those `const std::ostringstream` variables. They looked suspicious to me, because why would you declare a _const_ `std::ostringstream`?!? Then it appeared that you had recently added those `const` keywords (PR #5035 commit 375ee8c47a03792d0d3d5447c6b4c4be6b9f2fb4), because those two `std::ostringstream` variables were never modified! While they were originally _intended_ to been modified!!! 😸